Cargo.lock

```diff
@@ -4,3578 +4,3883 @@
 name = "account_manager"
 version = "0.0.1"
 dependencies = [
- "bls 0.2.0",
```

Comment:
The `Cargo.lock` has a huge diff. Interested if you did a `cargo update` or if cargo did this by itself.

Reply:
It was probably caused by me deleting the `Cargo.lock` file while resolving git conflicts.  I will try to base it on the latest `master`.